lib/osf-components/addon/components/registries/registration-form-navigation-dropdown/template.hbs

```diff
@@ -0,0 +1,72 @@
+<ResponsiveDropdown local-class='Dropdown' @renderInPlace={{true}} as |dd|>
+    {{#dd.trigger local-class='MenuTrigger'}}
+        <OsfButton
+            data-analytics-name='Expand'
+            data-test-toggle-anchor-nav-button @type='link'
+            @class={{local-class 'ToggleNav__button'}}>
+            <FaIcon @icon='bars' />
+        </OsfButton>
+    {{/dd.trigger}}
+    {{#dd.content}}
+        <div data-analytics-scope='Form navigation menu' local-class='NavigationMenu'>
+            <div local-class='Header' data-test-anchor-nav-title>
+                <div local-class='HeaderText'>{{t 'registries.overview.contents'}}</div>
+                <OsfButton
+                    data-analytics-name='Collapse'
+                    data-test-toggle-anchor-nav-button
+                    @onClick={{action dd.close}}
+                    @type='link' @class={{local-class 'ToggleNav__button'}}>
+                    <FaIcon @icon='times' />
+                </OsfButton>
+            </div>
+            <div local-class='Anchors' data-test-form-block-anchors>
+                {{#each this.blocksWithAnchor as |block|}}
+                    {{#if (eq block.blockType 'page-heading')}}
```

Comment:
all these block-type blocks are almost identical -- might be cleaner to move them into a subcomponent that has computeds for `local-class` and such